pkg/storage/unified/sql/backend.go

```diff
@@ -679,7 +687,7 @@ func (b *backend) poll(ctx context.Context, grp string, res string, since int64,
 		nextRV = rec.ResourceVersion
 		prevRV := rec.PreviousRV
 		if prevRV == nil {
-			*prevRV = int64(0)
+			prevRV = new(int64)
```

Comment:
Not related to the changes, but this is a `nil` pointer deref and will `panic`: https://go.dev/play/p/lRusspwG7a_c